src/sdp/SDP.ts

```diff
@@ -84,4 +85,9 @@ export default interface SDP {
    * Removes H.264 from the send section.
    */
   removeH264SupportFromSendSection(): SDP;
+
+  /**
+   * List of parsed media sections sections in order they occur on SDP.
+   */
+  mediaSections(): SDPMediaSection[];
```

Comment:
So this is a stretch as I doubt anyone would implement their own SDP but shouldn't this be optional :D